src/server/cli/load-config-file.ts

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 import { Command } from "commander";
-import { loadFileSync } from "../utils/file/file";
+import { loadJsonFileSync } from "../utils/file/file";
 
 export function loadConfigFile(configPath: string, program: Command): object {
   try {
-    return loadFileSync(configPath, "yaml");
+    return loadJsonFileSync(configPath);
```

Comment:
> [in] loadFileSync [..] removed second parameter as it is called always with "json" parameter

I mean there is a call with "yaml" flag. And configs are written in yaml always ...

Reply:
sorry, I'm blind